Unquote/Assertions.fs

```diff
@@ -62,6 +62,39 @@ module Internal =
 #if PORTABLE
         outputReducedExprsMsg outputGenericTestFailedMsg
 #else
+#if NETSTANDARD2_0

```

Comment:
@alex-bogomaz is there are reason for limiting the set of supported frameworks for NetStandard 2.0 here?

Reply:
The only reason was NetStandard support for each particular framework:
 - NUnit 3, Xunit 2 and Expecto - they support NetStandard 1.6.
 - Fuchu, Xunit 1, MBUnit - not
 - FSI - according to official MS announcements - it is not supported for NetCore yet: https://blogs.msdn.microsoft.com/dotnet/2017/08/14/f-and-net-core-roadmap-update/

Reply:
@alex-bogomaz thanks, that makes sense. But I wonder if it matters that Fuchu, Xunit 1, MBUnit, and FSI don't support NetStandard since we are loading assemblies dynamically... I forget why we had to limit the PCL build to only the generic test failed function.

Reply:
@stephen-swensen `Fuchu` is replaced by `Expecto` afaik.

Reply:
good point. maybe we should not limit. 
i'll test following scenario and get back with results:
- use dot net sdk 2.0
- create unit-test project, target net framework 461 
- reference netstandard 2.0 unquote
- reference full-framework Fuchu